frontend/src/components/Status/index.tsx

```diff
@@ -48,7 +48,9 @@ export default component$((props: Props) => {
 				<div class="leading-relaxed status-content" dangerouslySetInnerHTML={status.content} />
 			</div>
 
-			{status.media_attachments.length > 0 && <Image mediaAttachment={status.media_attachments[0]} />}
+			{status.media_attachments.map((attachment) => (
```

Comment:
have you checked how it looks with multiple media?

Reply:
Chatted offline